src/test/scala/akka/persistence/cassandra/CassandraLifecycle.scala

```diff
@@ -27,21 +27,24 @@ object CassandraLifecycle {
     akka.actor.serialize-messages=on
     """)
 
-  def awaitPersistenceInit(system: ActorSystem): Unit = {
+  def awaitPersistenceInit(system: ActorSystem, journalPluginId: String="", snapshotPluginId: String=""): Unit = {
     val probe = TestProbe()(system)
     val t0 = System.nanoTime()
     var n = 0
     probe.within(45.seconds) {
       probe.awaitAssert {
         n += 1
-        system.actorOf(Props[AwaitPersistenceInit], "persistenceInit" + n).tell("hello", probe.ref)
+        system.actorOf(Props(classOf[AwaitPersistenceInit], journalPluginId, snapshotPluginId), "persistenceInit" + n).tell("hello", probe.ref)
         probe.expectMsg(5.seconds, "hello")
         system.log.debug("awaitPersistenceInit took {} ms {}", TimeUnit.NANOSECONDS.toMillis(System.nanoTime() - t0), system.name)
       }
     }
   }
 
-  class AwaitPersistenceInit extends PersistentActor {
+  class AwaitPersistenceInit(
+    override val journalPluginId:  String,
+    override val snapshotPluginId: String
+  ) extends PersistentActor {
```

Comment:
Right, so previously on start of each test we call 
`CassandraLifecycle.awaitPersistenceInit(system)`
This creates PersistentActor and waits until actor is able to respond with message that received.
And it works fine with all other tests except `MultiPluginSpec` where this functionality was disabled because PersistentActor uses default cassandra-journal unless explicitly said.
And it this suite of tests default cassandra-journal doesn't exist that's why this actor couldn't work here.
But if explicitly said what cassandra-journal actor should operate on we can also use this awaiting step in `MultiPluginSpec` tests.

if `cassandra-journal` / `snapshot-journal` is set to "" then default one is used that's why by default it is empty string